core/testcontainers/core/config.py

```diff
@@ -49,7 +50,6 @@ class TestcontainersConfiguration:
     ryuk_image: str = RYUK_IMAGE
     ryuk_privileged: bool = RYUK_PRIVILEGED
     ryuk_disabled: bool = RYUK_DISABLED
-    ryuk_docker_socket: str = RYUK_DOCKER_SOCKET
```

Comment:
we need to leave this configurable because otherwise people want to monkeypatch our modules

Reply:
Can't they just use the environment variable to configure it if someone ever needs it? A [quick search on GitHub](https://github.com/search?q=TestcontainersConfiguration%28+language%3APython&type=code&l=Python) shows no-one is actually calling `TestcontainersConfiguration` with custom params

Otherwise we can keep `c.ryuk_docker_socket` as a direct property for people to configure

And rename the new computed property to something else, e.g. `c.docker_socket` or `c.get_docker_socket()` (and use this computed property when needing to provide the docker socket)

Reply:
see #531

Reply:
I don't really see where is the problem, it is possible to monkeypatch a computed property:

```python
from testcontainers.core.config import testcontainers_config, TestcontainersConfiguration

def custom_ryuk_docker_socket(self) -> str:
    return "/custom/path/to/docker.sock"

TestcontainersConfiguration.ryuk_docker_socket = property(custom_ryuk_docker_socket)
print(testcontainers_config.ryuk_docker_socket)
```

Otherwise I proposed another solution with a static property + a computed one, would this fit?